test/bin/build_images.sh

```diff
@@ -706,6 +706,11 @@ fi
 
 trap 'osbuild_logs' EXIT
 
+# Check if webserver is running
+if ! ps aux | grep nginx | grep -v grep > /dev/null; then
```

Comment:
This might be a more effective way to check:
```
if [ $(pgrep -cx nginx) -eq 0 ] ; then
fi
```

Reply:
yes, I just ran shellcheck and it suggested the same thing so I'm gonna do that, thanks :)